src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)coreclr.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)dbgshim.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordaccore.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.debug.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)System.Private.CoreLib.ni.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.debug.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)System.Private.CoreLib.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll" />
```

Comment:
I was wondering about matching the dll/exe directory especially after adding them back to the symbol package. So it doesn’t matter?

Archive:  runtime.win7-x64.microsoft.netcore.runtime.coreclr.1.0.3-rc4-24221-0.symbols.nupkg
  Length      Date    Time    Name
---------  ---------- -----   ----
      544  06/21/2016 14:32   _rels/.rels
     1271  06/21/2016 14:32   runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR.nuspec
     9338  06/21/2016 14:32   dotnet_library_license.txt
     1617  06/21/2016 14:32   ThirdPartyNotices.txt
        0  06/21/2016 14:32   ref/netstandard1.0/_._
        0  06/21/2016 14:32   runtimes/win7-x64-aot/lib/netstandard1.0/_._
        0  06/21/2016 14:32   runtimes/win7-x64-aot/native/_._
  2307584  06/21/2016 14:32   runtimes/win7-x64/lib/netstandard1.0/System.Private.CoreLib.dll
    30720  06/21/2016 14:32   runtimes/win7-x64/lib/netstandard1.0/mscorlib.dll
    33280  06/21/2016 14:32   runtimes/win7-x64/native/mscorlib.ni.dll
10802688  06/21/2016 14:32   runtimes/win7-x64/native/System.Private.CoreLib.ni.dll
   223232  06/21/2016 14:32   runtimes/win7-x64/native/clretwrc.dll
  5286400  06/21/2016 14:32   runtimes/win7-x64/native/coreclr.dll
   135168  06/21/2016 14:32   runtimes/win7-x64/native/dbgshim.dll
  1754624  06/21/2016 14:32   runtimes/win7-x64/native/mscordaccore.dll
  1424384  06/21/2016 14:32   runtimes/win7-x64/native/mscordbi.dll
   355328  06/21/2016 14:32   runtimes/win7-x64/native/mscorrc.debug.dll
     4608  06/21/2016 14:32   runtimes/win7-x64/native/mscorrc.dll
   691200  06/21/2016 14:32   runtimes/win7-x64/native/sos.dll
    86016  06/21/2016 14:32   runtimes/win7-x64/native/clretwrc.pdb
55316480  06/21/2016 14:32   runtimes/win7-x64/native/coreclr.pdb
  3272704  06/21/2016 14:32   runtimes/win7-x64/native/dbgshim.pdb
28495872  06/21/2016 14:32   runtimes/win7-x64/native/mscordaccore.pdb
15667200  06/21/2016 14:32   runtimes/win7-x64/native/mscordbi.pdb
    86016  06/21/2016 14:32   runtimes/win7-x64/native/mscorrc.debug.pdb
    86016  06/21/2016 14:32   runtimes/win7-x64/native/mscorrc.pdb
  5500928  06/21/2016 14:32   runtimes/win7-x64/native/sos.pdb
  6143488  06/21/2016 14:32   runtimes/win7-x64/native/System.Private.CoreLib.pdb
     7680  06/21/2016 14:32   runtimes/win7-x64/native/mscorlib.pdb
27021312  06/21/2016 14:32   runtimes/win7-x64/native/crossgen.pdb
        0  06/21/2016 14:32   src/_._
  2765824  06/21/2016 14:32   tools/crossgen.exe
      715  06/21/2016 14:32   [Content_Types].xml
     1017  06/21/2016 14:32   package/services/metadata/core-properties/e22c41fc2863404dbf257ea26029949e.psmdcp
---------                     -------
167513254                     34 files